### DISCUSSION TITLE
Hyperparameter Tuning For SNNs

### DISCUSSION THREAD

**Discussion by a2l5t8** [cnrl/CoNeX#63] [Q&A]
Hi, 
I am looking for an efficient way to fine-tune hyperparameters of my network in CoNeX, is there any framework or specific algorithm help achieve this goal instead of simple grid search or manually swapping out different parameter values?

Thanks in advance.

**Comment by realamirhe** [comment]
[optuna](https://github.com/optuna/optuna) seems to be a good fit as a hyperparameter optimization framework
